.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
```

Comment:
i guess i can remove this stage since build & push action will use git context, and it's seems to be the branch that we checkout. (https://github.com/marketplace/actions/build-and-push-docker-images#usage)

Reply:
no problem if you need to remove it as long as the last step still works